test/suite/run.py

```diff
@@ -178,16 +179,38 @@ def show_env(verbose, envvar):
 # e.g. test_util03 -> util
 reCatname = re.compile(r"test_([^0-9]+)[0-9]*")
 
+# Look for a list of the form 0-9,11,15-17.
+def parse_int_list(str):
+    ret = {}
+    # divide into ranges separated by commas
```

Comment:
@sauclovian-wt all comments should be fully formed sentences.

Reply:
Right, I fixed the rest of them. Sorry, this is probably going to continue to happen some in the short term as I'm not used to that requirement :-/